wpilibc/src/main/native/cpp/smartdashboard/SendableBuilderImpl.cpp

```diff
@@ -88,7 +90,8 @@ void SendableBuilderImpl::AddBooleanProperty(const wpi::Twine& key,
       return entry.AddListener(
           [=](const nt::EntryNotification& event) {
             if (!event.value->IsBoolean()) return;
-            setter(event.value->GetBoolean());
+            SmartDashboard::PostListenerTask(
+                [&setter, &event] { setter(event.value->GetBoolean()); });
```

Comment:
This is unsafe.  You have to copy the event, and the setter as well, so you might as well use =.